### PR TITLE
fix: readdress scripting removal with chart upgrade fix

### PR DESCRIPTION
**Pull request by deanesmith**
Second attempt now that chart upgrades are working.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
-FROM sonatype/nexus3:3.20.1
+FROM sonatype/nexus3:3.22.1
 
 COPY *.json /opt/sonatype/nexus/
-COPY repositories /opt/sonatype/nexus/repositories
+COPY maven-proxy-repositories /opt/sonatype/nexus/maven-proxy-repositories
+COPY npmjs-proxy-repositories /opt/sonatype/nexus/npmjs-proxy-repositories
```
Comment (abayer) [inline Dockerfile:5]: Don't we need to copy the `disable-anonymous-access.json` file too?

File: Dockerfile [hunk 2]
```diff
@@ -1,12 +1,12 @@
-FROM sonatype/nexus3:3.20.1
+FROM sonatype/nexus3:3.22.1
 
 COPY *.json /opt/sonatype/nexus/
```
Comment (abayer) [inline Dockerfile:3]: ah, ok, it is getting copied. I misread.

**Comment by deanesmith** [comment]
Should address https://github.com/jenkins-x-charts/nexus/issues/54.

**Pull request merged**